uber/models.py

```diff
@@ -1938,6 +1953,22 @@ class ArbitraryCharge(MagModel):
     _repr_attr_names = ['what']
 
 
+class LedgerItem(MagModel):
+    ident = Column(UnicodeText, nullable=True)
+    type = Column(Choice(c.LEDGER_TYPE_OPTS), default=c.ACCOUNTS_RECEIVED)
```

Comment:
FWIW, I frequently use ``type`` as a column name; the codebase I was working on today at work has a SQLAlchemy ``type`` column.

``type`` is a builtin function but not a reserved word, so it's fine to use in this context.  It won't even cause problems because defining it as a column property like this won't overwrite the global ``type`` definition or anything crazy like that.  (For the same reason I'm fine with using ``id`` as a parameter name in a function even though that's also a builtin method.)